src/Shell/Task/FixtureTask.php

```diff
@@ -386,6 +387,9 @@ protected function _makeRecordString($records)
         foreach ($records as $record) {
             $values = [];
             foreach ($record as $field => $value) {
+                if ($value instanceof DateTime) {
```

Comment:
This is still an issue when using immutable time objects (specified in bootstrap.php).  `$value` will be an instance of `Cake\I18n\FrozenTime`.

 Should this `if` statement check if `$value` is an instance of `DateTimeInterface` instead of `DateTime`?

This resolved the issue for me.